### PR TITLE
Submitting EMX Digital Adapter

### PR DESCRIPTION
**Pull request by EMXDigital**
## Type of change
- [x] New bidder adapter

## Description of change

Submitting standalone EMX Digital adapter for Prebid.

- test parameters for validating bids
```
{
  bidder: 'emx_digital',
  params: {
        tagid: '25251'
  }
}
```

- contact email of the adapter’s maintainer
git@emxdigital.com

- [x] official adapter submission

### CODE REVIEW

**Comment by bretg** [comment]
Docs PR https://github.com/prebid/prebid.github.io/pull/992

File: modules/emx_digitalBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+          netRevenue: true,
+          mediaType: BANNER,
+          ad: decodeURIComponent(emxBid.adm),
+          ttl: emxBid.ttl
+        });
+      });
+    }
+    return emxBidResponses;
+  },
+  getUserSyncs: function (syncOptions) {
```
Comment (jaiminpanchal27) [inline modules/emx_digitalBidAdapter.js]: This method is optional. You can remove since you are not using

File: modules/emx_digitalBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      method: 'POST',
+      url: url,
+      data: JSON.stringify(emxData),
+      options: {
+        withCredentials: true
+      }
+    };
+  },
+  interpretResponse: function (serverResponse) {
+    var emxBidResponses = [];
```
Comment (jaiminpanchal27) [inline modules/emx_digitalBidAdapter.js]: Use let/const everywhere.

**Comment by jaiminpanchal27** [comment]
@EMXDigital Thanks for updates. Everything looks good. 
Could you rebase of current master so that circleci would pass all the tests.

**Pull request merged**